### PR TITLE
ci: fix lint-fmt job

### PR DESCRIPTION
**Pull request by tatchi**
It currently fails to parse the `ocamlformat` version due to the comment added in https://github.com/ocamllabs/vscode-ocaml-platform/pull/746

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -161,6 +161,6 @@ jobs:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
           dune-cache: true
 
-      - run: opam depext ocamlformat=$(cat .ocamlformat | grep version | cut -d '=' -f 2) --install
+      - run: opam depext ocamlformat=$(cat .ocamlformat | grep version | cut -d '=' -f 2 | cut -d ' ' -f 1) --install
```
Comment (smorimoto) [inline .github/workflows/main.yml:164]:
```suggestion
      - run: opam depext --install ocamlformat=$(awk -F = '/version/{print $2}' .ocamlformat)
```

Comment (smorimoto) [reply .github/workflows/main.yml:164]: How about simply moving the comment to a line above and replacing the step with this?

Comment (smorimoto) [reply .github/workflows/main.yml:164]: There is not much difference, but I'm avoiding UUOC these days.
https://en.wikipedia.org/wiki/Cat_(Unix)#Useless_use_of_cat

Comment (tatchi) [reply .github/workflows/main.yml:164]: I'm not familiar with Unix commands in general so I fully trust you on that 😄 Thanks for the suggestion :)

**Review by smorimoto — APPROVED** [review]

**Comment by smorimoto** [comment]
Oh no please do not merge this.

**Comment by smorimoto** [comment]
Okay, I think this is now ready for merging.

**Comment by tatchi** [comment]
Thanks for the review @smorimoto 🤗

**Pull request merged**